Riot/Modules/Settings/SettingsViewController.m

```diff
@@ -315,13 +314,9 @@ @interface SettingsViewController () <UITextFieldDelegate, MXKCountryPickerViewC
 
 @property (nonatomic, strong) UserInteractiveAuthenticationService *userInteractiveAuthenticationService;
 
-<<<<<<< HEAD
 // Tchap: Threads are disabled
 //@property (nonatomic, strong) ThreadsBetaCoordinatorBridgePresenter *threadsBetaBridgePresenter;
-=======
-@property (nonatomic, strong) ThreadsBetaCoordinatorBridgePresenter *threadsBetaBridgePresenter;
 @property (nonatomic, strong) ChangePasswordCoordinatorBridgePresenter *changePasswordBridgePresenter;
```

Comment:
I think you don't use this delegate

Reply:
Yes, it's not used. I will comment it.

Reply:
Sorry for my previous comment, but the `ChangePasswordCoordinatorBridgePresenter` is needed for the change password feature.